Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -290,150 +290,152 @@ - (void)stopLoading {
 }
 
 - (HRESULT)_asyncHttpOperation:(AsyncHttpOperation*)pOperation completedWithStatus:(AsyncStatus)status {
-    if (status == AsyncStatus::Canceled) {
-        return S_OK;
-    }
-
-    NSError* error = nil;
-    try {
-        ComPtr<AsyncHttpOperation> operation(pOperation);
-        ComPtr<IAsyncInfo> asyncInfo;
-        THROW_IF_FAILED(operation.As(&asyncInfo));
-
-        if (status == AsyncStatus::Error) {
-            HRESULT hr = S_OK;
-            THROW_IF_FAILED(asyncInfo->get_ErrorCode(&hr));
-            THROW_HR(hr);
-        } else if (status != AsyncStatus::Completed) {
-            THROW_HR(E_UNEXPECTED);
+    @autoreleasepool {
```

Comment:
>@autoreleasepool { [](start = 4, length = 18)

Is there something more general we should do to make sure our threads have pools set up?